md/message-states.md

```diff
@@ -1,6 +1,27 @@
-# Message states
+# SMS message states
 
-All messages in the system are in one of the following states.
+## Interaction between medic-webapp and medic-gateway
+[Medic-webapp](https://github.com/medic/medic-webapp) uses [medic-gateway](https://github.com/medic/medic-gateway)
+to send SMS messages.
+
+When a report comes in, [medic-sentinel](https://github.com/medic/medic-sentinel) adds the appropriate list of
+scheduled messages (to be sent at a future date) to the report doc.
+
+Periodically, sentinel checks for messages that need to be sent, and [sets their status to `pending`](https://github.com/medic/medic-sentinel/blob/master/schedule/due_tasks.js).
```

Comment:
> sets their status to `pending`

...if their scheduled sending time has been reached or passed.

Also, in medic-webapp/-sentinel/-api, messages have a **_state_** rather than a **_status_**.